accounts/views.py

```diff
@@ -0,0 +1,27 @@
+from django.contrib.auth import authenticate, login, logout
+from django.http import HttpRequest, HttpResponse, HttpResponseRedirect
+from django.shortcuts import render
+from django.urls import reverse
+
+
+# Create your views here.
+def login_view(request: HttpRequest) -> HttpResponse:
+    if request.method == "POST":
+        username = request.POST["username"]
+        password = request.POST["password"]
+        user = authenticate(request, username=username, password=password)
+        if user is not None:
+            login(request, user)
+            return HttpResponseRedirect(reverse("taxi:index"))
```

Comment:
Ensure that the URL name 'taxi:index' used in `reverse()` is correctly defined in your URL configurations. If 'taxi:index' does not exist, it will cause a `NoReverseMatch` error.